### ISSUE TITLE
The test assertion that the primary node should be the first element in the list of nodes fails

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#777]
Originally reported of Google Code with ID 721.

```
<b>What steps will reproduce the problem?</b>
1. Modify the test in testNodesDrbdDisks so that the primary node isn't the first in alphabetical order
2. Run PYTHONPATH=.:test/py ./test/py/ganeti.objects_unittest.py

<b>What is the expected output? What do you see instead?</b>

The test should pass. It fails instead.


```
Originally added on 2014-02-18 14:47:27 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
The test was added in 6a05000766e72e999c6991f1de7b2ed38f5c3a16 about 2 years after the all_nodes attribute was introduced, and it's not clear why it it requires the primary node to be the first node.
```
Originally added on 2014-02-18 15:11:54 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in

commit f2a3c4f07f1991d9a2ab0657c3251812361a239c
Author: Petr Pudlak <pudlak@google.com>
Date:   Mon Feb 24 09:27:57 2014 +0100

    Let the instance's tuple of nodes start with the primary
    
    Before the tuple of nodes of an instance was created from a set, listing
    the nodes in alphabetical order. This patch ensures that the primary
    node is always the first one in the list.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Jose A. Lopes <jabolopes@google.com>

commit 93a968aac088046832d037806afc5e7433cd712f
Author: Petr Pudlak <pudlak@google.com>
Date:   Mon Feb 24 09:26:08 2014 +0100

    Fix the test that checks for the order of instance's nodes
    
    The test checks if the first node in the tuple is the primary node, but
    because it names the node so that it's the least one alphabetically, the
    test always succeeds. This fixes the test.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Jose A. Lopes <jabolopes@google.com>

```
Originally added on 2014-02-24 09:21:49 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in 2.10.1
```
Originally added on 2014-03-05 13:55:53 +0000 UTC.

Changed State: Released
